tests/JSValueTests.Usage.swift

```diff
@@ -135,11 +135,11 @@ class JSValueUsageTests : XCTestCase {
                 "url" : "http://remote.bloxus.com/"
             ]
         
-        let blog = make ⇒
-            (json["id"].number ⇒ toInt) ⇒
-            json["name"].string ⇒
-            json["needspassword"].bool ⇒
-            (json["url"].string ⇒ toURL)
+        let blog = make(
```

Comment:
Yeah, so that means this decl:

```swift
func make(id: Int?)(name: String?)(needsPassword: Bool?)(url: NSURL?) -> Blog?
```

Has to change to this:

```swift
func make(id: Int?) -> (name: String?) -> (needsPassword: Bool?) -> (url: NSURL?) -> Blog?
```

And then the body of the function has to get updated as the example in the spec:

```swift
  // Before:
  func curried(x: Int)(y: String) -> Float {
    return Float(x) + Float(y)!
  }

  // After:
  func curried(x: Int) -> (String) -> Float {
    return {(y: String) -> Float in
      return Float(x) + Float(y)!
    }
  }```